publisher/guides/api-usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: How to use the Rest API
+---
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## WordPress application password
+
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not use for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client 
+application. If a client application compromise, the user can revoke the password specifically for this application.
+
+### Create an application password
+
+<img src="/assets/publisher/api-users.png" />
+
+WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account. Access is through 
+the user profile page.
+
+<img src="/assets/publisher/api-application-password-create.png" />
+
+It is important to save the now visible password securely right away. After leaving the page, the password is no longer visible and 
+cannot be displayed again. In the list all created Application Passwords are listed and can revoke individually.
+
+<img src="/assets/publisher/api-application-password-list.png" />
+
+### Test der API mit Postman
+
+The generation of an episode exemplifies the usage.
+
+1. Select HTTP method POST
+2. Enter the URL http://podlove.local/wp-json/podlove/v2/episodes
+3. Select the Authorization tab
+4. Select Basic Auth as type
+5. Enter username and application password
+6. Press the Send button
+7. The response (status and body) appears in the lower part of the screen
+
```

Comment:
Nicely written :) Very cool guide for our API uses.  Let me know when it is live so we can tweet about it :)